src/content/community/releases/_index.en.md

```diff
@@ -4,6 +4,40 @@ title = "Releases"
 weight = 40
 +++
 
+## v0.9.0
+
+* The gateway Pod has been renamed from `submariner` to `submariner-gateway`.
+* The Helm charts now support Submariner deployments using the Operator.
+* Broker creation is now managed by the Operator (and not `subctl`).
+* Each Submariner Pod now has its own service account with appropriate privileges.
+* For OpenShift, the Lighthouse CoreDNS server metrics are now included with the default CoreDNS metrics.
+* The `submariner_connections` metric is renamed to `submariner_requested_connections`.
+* For cases in which cross-cluster connectivity is provided without Submariner, `subctl` can now just deploy
+  Service Discovery via the `--components` flag.
+* Improved Service CIDR discovery for k3s deployments.
+* All Submariner metrics are now prefixed with `submariner_` in Prometheus.
+* With Globalnet deployments, Global IPs are now assigned to exported Services only. Previously, Globalnet annotated
+  every Service in the cluster, whether or not it was exported.
+* Lighthouse now allows a user to set name of CoreDNS custom ConfigMap.
+* The `strongswan` cable driver which was deprecated in v0.8.0 release is now removed.
+* Configuring Globalnet on a cluster is now optional with the `--globalnet` flag.
+* The Lighthouse-specific API is now removed in favor of [Kubernetes Multicluster Services API](https://github.com/kubernetes-sigs/mcs-api/).
+* A diagnostic tool [`subctl validate`](https://submariner.io/operations/deployment/subctl/#validate) was added to detect issues with the
+  Submariner deployment or some configurations in the cluster that may prevent Submariner from working properly.
+* Provision to notify user if `subctl` and `Submariner` versions are not matched in a setup was added.
+* Multiple new make-clean targets, `make clean-generated`, `make clean-clusters`and `make clean`, were added.
```

Comment:
This is not user-facing

```suggestion
```

Reply:
Users are using `make` so I guess they should be aware of new features added?

Reply:
I agree with Tom here. The typical user should not care about kind other then very basic things (basically the quickstart guide).

Reply:
`make` is for devs - this guide is for end-users of the releases product. This kind of note would go into release notes on the specific git project.